packages/lexical-playground/src/nodes/ImageNode.tsx

```diff
@@ -52,6 +53,7 @@ function convertImageElement(domNode: Node): null | DOMConversionOutput {
 export type SerializedImageNode = Spread<
   {
     altText: string;
+    title: string;
```

Comment:
It's a new field so it's not guaranteed to be there in currently serialized nodes
```suggestion
    title?: string;
```